### PR TITLE
Enable travis-ci #119

### PR DESCRIPTION
**Pull request by gianarb**
* Add travis-ci configuration

### CODE REVIEW

**Comment by snapshotpl** [comment]
You missed php 5.3

**Comment by gianarb** [comment]
@ezimuel do we support php 5.3?

**Comment by ezimuel** [comment]
@gianarb yes, the requirement for ZFTool is PHP >= 5.3.3.

**Comment by gianarb** [comment]
Ok I add this check in travis and I remove short array syntax from phpcs

**Comment by snapshotpl** [comment]
Hey guys! Can we finally enable CI?

**Comment by gianarb** [comment]
@snapshotpl have you idea about this error?
```
 You must enable the openssl extension in your php.ini to load information from https://packages.zendframework.com          
```
https://travis-ci.org/zendframework/ZFTool/jobs/74531550

**Comment by weierophinney** [comment]
@gianarb You need to have a PHP version that has openssl installed, essentially. Additionally, older versions of PHP have broken SSL versions that leave users vulnerable to certain security exploits, and Composer tries to detect those before attempting to send a request.

I would argue you should remove the 5.3.3 entry for travis, and use simply "5.3" if you want to continue to test on 5.3 at this time; we raised the minimum 5.3 version to 5.3.23 with ZF 2.2, IIRC, so that should be safe.

**Comment by gianarb** [comment]
@weierophinney I removed new a repository configuration from composer.json becuase it is out of date IMO.. Maybe it resolve the problem :)

**Comment by snapshotpl** [comment]
@gianarb Let's merge everything! :-)

**Comment by snapshotpl** [comment]
@gianarb to remove php 5.5 failure remove `--ignore-platform-reqs`

**Comment by gianarb** [comment]
Already resolved it :) Thanks!!

2015-09-02 17:51 GMT+02:00 Witold Wasiczko <notifications@github.com>:

> @gianarb <https://github.com/gianarb> to remove php 5.5 failure remove
> --ignore-platform-reqs
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/ZFTool/pull/123#issuecomment-137141241>.
>



-- 
Gianluca Arbezzano
www.gianarb.it

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+sudo: false
+
+language: php
+
+matrix:
```
Comment (Maks3w) [inline .travis.yml]: Please cache composer dependencies

https://github.com/zfcampus/zf-apigility-skeleton/blob/master/composer.json#L36

**Comment by Maks3w** [comment]
@gianarb Please remove `--ignore-platform-reqs`

**Comment by Maks3w** [comment]
Please enable test coverage too https://github.com/Maks3w/zend-expressive/commit/225e40d6914fbb8c0680c2e048015b3b39e35188

**Comment by gianarb** [comment]
`unknown repository` I removed it sorry..

**Comment by Maks3w** [comment]
You can still retrieve your work https://patch-diff.githubusercontent.com/raw/zendframework/ZFTool/pull/123.patch

**Comment by gianarb** [comment]
thanks

**Pull request closed without merging**